File: C#/Rajyasabha.cs

class RajyaSabha {
    private Chairman chairman;
    private MemberOfParliament[] members;
    private int counter = 0;


    public RajyaSabha(Chairman chairman){
        this.chairman = chairman;
        members = new MemberOfParliament[250];
    }

    public void passBill(Bill bill) {
        Console.WriteLine("Rajya Sabah is passing bill :\n"+bill.getBill());
    }

    public void addMember(MemberOfParliament member){
        if(counter<250){   
            members[counter] = member;
            counter++;
            Console.WriteLine("Memeber: "+member.getMPDetails()+ " added to Rajya Sabha...");
        }
        else{
            Console.WriteLine("Rajya Sabha is full...");
        }
    }

    public string getChairman(){
        string sp = this.chairman.getMPDetails();
        return sp;
    } 

    public MemberOfParliament[] getMPsRajyaSabha(){
        return this.members;
    }
}

File: PY/rajyasabha.py
# Chairman

class RajyaSabha:
    
    def __init__(self, chairman):
      self.chairman = chairman
      self.members = []
      
    def pass_bill(self,bill):
        print(f"Bill {bill.title} passed by the Rajya Sabha.\n")
        
    def add_members(self, member):
        if(len(self.members)<543):
            self.members.append(member)
            print(f"{member.get_mp_details()} added to Rajya Sabha..\n")
        else:
            print("Rajya Sabha full..\n")
            
    def get_chairman(self):
        return self.chairman.get_mp_details()
    
    def get_mps_rajya_sabha(self):
        return self.members